sdks/python/apache_beam/runners/portability/expansion_service_test.py

```diff
@@ -270,6 +270,20 @@ def from_runner_api_parameter(unused_ptransform, payload, unused_context):
     return PayloadTransform(payload.decode('ascii'))
 
 
+@ptransform.PTransform.register_urn('sum_without_type', None)
+class SumWithoutTypeTransform(ptransform.PTransform):
+  def expand(self, pcoll):
+    return pcoll | beam.CombineGlobally(sum)
```

Comment:
Can we never infer this? Better to use some type that we know we'll never be able to infer (e.g. due to a data dependency like "Map(lambda x: str_literal if x else int_literal)" that would technically be a union type but we could ensure is not).

Reply:
Changed the test transform.